### DISCUSSION TITLE
请问全局开启审计是否会对doris造成比较大的性能问题

### DISCUSSION THREAD

**Discussion by stgztsw** [apache/doris#11553] [A - General / Q&A]
请问全局开启审计是否会对doris造成比较大的性能问题
set global enable_partition_cache=true;

**Comment by hf200012** [comment]
set global enable_partition_cache=true;
这个是开启partition缓存，只会获得更好的效果
审计日志是个插件，不影响性能，你也可以通过fe.audit.log获取审计日志

**Reply by stgztsw** [reply]
写错了，是set global enable_profile=true；  主要是想找到慢查询的sql

**Reply by dataalive** [reply]
这个还是会影响一部分性能的，你查找慢SQL的话，直接看audit log？还是已经知道哪个sql慢了分析下为啥慢？这种可以session级别开启即可。
